ios/browser/application_context/brave_application_context_impl.mm

```diff
@@ -111,4 +126,10 @@
 
   brave_wallet::WalletDataFilesInstaller::GetInstance().SetDelegate(
       std::make_unique<brave_wallet::WalletDataFilesInstallerDelegateImpl>());
+
+  leo_local_models_updater();
```

Comment:
why this is not under a feature check?

Reply:
With W7e443e164c9e4952e57e1d040e9f300d7fa521e2, we don't need separate handling in iOS.